provider/local/release.go

```diff
@@ -53,6 +54,9 @@ func (p *Provider) ReleaseGet(app, id string) (*types.Release, error) {
 	var r *types.Release
 
 	if err := p.storageLoad(fmt.Sprintf("apps/%s/releases/%s/release.json", app, id), &r, ReleaseCacheDuration); err != nil {
+		if strings.Contains(err.Error(), "no such key") {
```

Comment:
Use awsError(err) to get an aws standard "code" to detect the error rather than going off error text